app/models/alchemy/page.rb

```diff
@@ -361,6 +361,34 @@ def update_node!(node)
       update_columns(hash)
     end
 
+
+    # Returns true if the page cache control headers should be set.
+    #
+    # == Disable Alchemy's page caching globally
+    #
+    #     # config/alchemy/config.yml
+    #     ...
+    #     cache_pages: false
+    #
+    # == Disable caching on page layout level
+    #
+    #     # config/alchemy/page_layouts.yml
+    #     - name: contact
+    #       cache: false
+    #
+    # Note: This only sets the cache control headers and skips rendering of the page body, if the cache is fresh.
+    # This does not disable the fragment caching in the views. So if you don't want a page and it's elements to be cached,
```

Comment:
Would you mind to add proper line breaks to the comments. Not you're fault, I know, but as you are at here anyway?